README.md

```diff
@@ -13,40 +14,172 @@ In alphabetical order:
 |              [<img src="https://github.com/favicon.ico" width="15"> Github](https://github.com/alexbotello)              |                      [<img src="https://github.com/favicon.ico" width="15"> Github](https://github.com/ahrjarrett)                      |              [<img src="https://github.com/favicon.ico" width="15"> Github](https://github.com/JohnCoronel)              |                    [<img src="https://github.com/favicon.ico" width="15"> Github](https://github.com/iamthuypham)                    |               [<img src="https://github.com/favicon.ico" width="15"> Github](https://github.com/ujaved931)               |                 [<img src="https://github.com/favicon.ico" width="15"> Github](https://github.com/victoramontoya)                  |
 | [ <img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15"> LinkedIn](https://www.linkedin.com/in/) | [ <img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15"> LinkedIn](https://www.linkedin.com/in/andrewhjarrett/) | [ <img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15"> LinkedIn](https://www.linkedin.com/in/) | [ <img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15"> LinkedIn](https://www.linkedin.com/in/iamthuypham/) | [ <img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15"> LinkedIn](https://www.linkedin.com/in/) | [ <img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15"> LinkedIn](https://www.linkedin.com/in/vamontoya/) |
 
-## Quick Overview
+# Table of Contents
+
+Generate by running `Markdown Preview Enhanced: Create TOC`
+
+# Scripts
+
+## Testing
+
+`yarn test`: Run tests (Alex is this correct?)
```

Comment:
Since we only have testing for the server right now the script only works from the server directory.

From the server directory:
`yarn run test`

It would be good to add that running tests locally requires a local mongo instance to be running. To start a mongo instance run these commands:
`sudo service mongod stop` - This is will clear default port if any previous mongo instances were not properly shutdown
`sudo mongod --fork --syslog` - This will start a mongo instance running in the background

Reply:
Just saw this 👍  I'll try to update tonight